databricks/koala/structures.py

```diff
@@ -278,6 +278,32 @@ def unique(self):
         # Pandas wants a series/array-like object
         return _col(self.to_dataframe().unique())
 
+    @derived_from(pd.Series)
+    def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+        if bins is not None:
+            raise NotImplementedError("value_counts currently does not support bins")
+
+        if dropna:
+            if isinstance(self.schema[self.name].dataType, (FloatType, DoubleType)):
+                pred = ~(self._spark_isNull() | F._spark_isnan(self))
```

Comment:
You just implemented Series.isnull(). This is fine, we can refactor this piece later. I made an issue to do that later.